website/cue/reference/components/sinks/datadog_traces.cue

```diff
@@ -4,7 +4,7 @@ components: sinks: datadog_traces: {
 	title: "Datadog Traces"
 
 	classes: sinks._datadog.classes & {
-		stateful: true
```

Comment:
I think I added this stateful flag. It came out of one of my recent gardener rotations where a user had pointed out that some of the sinks were actually technically stateful but weren't documented as such.

Reply:
That's due to the apm stats feature?

Reply:
I _think_ we may want to reword the context that's included for state vs stateless. I can see how the stateless description is not totally wrong for traces 🤔

Reply:
Doh, thanks for catching, I didn't mean to drop that.

Reply:
Sorry, I agree - it was more of a observation as I read the descriptions.